OPHYKIKEH-123: Ilmoittautumisaika mukaan osallistujalistaukseen

pkoivisto opened the pull request.

pkoivisto committed:
Return the created timestamp for registration with exam session participants listing.

pkoivisto committed:
Try to force updating apt-get package information

pkoivisto commented on .github/workflows/yki.yml:
```diff
@@ -40,6 +40,8 @@ jobs:
 
       - name: Install wkhtmltopdf
         run: |
+          echo 'Updating apt package information'
+          sudo apt-get update
```
Seuraava vaihe `Installing wkhtmltopdf` failasi aiemmin tänään kun apt-getin pakettilistaus ei ollut ajan tasalla: https://github.com/Opetushallitus/yki/actions/runs/3892524036/jobs/6644065088

mikhuttu commented on resources/yki/queries.sql:
```diff
@@ -1058,6 +1058,7 @@ AND state = 'COMPLETED';
 
 -- name: select-exam-session-participants
 SELECT
+  r.created,
```
Tän vois varmaan lisätä myös yki.spec:iin `::exam-session-participant` alle

pkoivisto replied:
Lisään. Päivitän vielä tuon muutoksen johdosta hajonneen testin.

pkoivisto committed:
Review feedback: add returned attribute to spec

pkoivisto committed:
Update test: assert created timestamp is included in response data.

Cannot just modify fixture, as creation time is dependent on when test is run and thus massaging
the fixture before comparison.
Another option would of course be to use a fixed time for test runs, but that would be a more involved change.
